src/ekomark/benchmark/runner.py

```diff
@@ -248,7 +249,13 @@ def log(self, theory, _, pdf, me, ext):
         for q2 in q2s:
             log_tab = dfdict.DFdict()
             ref_pdfs = ext["values"][q2]
-            res = pdf_grid[q2]
+            # find the first hit, regardless of the flavor, since others can not deal with them appearing in parallel
```

Comment:
That's a bit sad, but of course correct...
(unless we start tricking them with multiple runs, but it could be a pain, definitely not for this PR)

Reply:
> definitely not for this PR

agreed

Reply:
you mean the python comment? here we are comparing eko output against others (e.g. APFEL); now we can have a single Q2 have appear twice: `(4.92,4)` and `(4.92,5)` ... I have to check I'm not running in this situation and if I'm not there is only one hit and that is the one

Reply:
Even 4 times :P
(i.e. even `(4.92, 3)` and `(4.92, 6)` are possible, in principle)

Reply:
I understand, Thanks!